src/rpc/client/ts/package-lock.json

```diff
@@ -78,7 +78,7 @@
     "@protobufjs/aspromise": {
       "version": "1.1.2",
       "resolved": "https://registry.npmjs.org/@protobufjs/aspromise/-/aspromise-1.1.2.tgz",
-      "integrity": "sha512-j+gKExEuLmKwvz3OgROXtrJ2UG2x8Ch2YZUxahh+s1F2HZ+wAceUNLkvy6zKCPVRkU++ZWQrdxsUeQXmcg4uoQ==",
+      "integrity": "sha1-m4sMxmPWaafY9vXQiToU00jzD78=",
```

Comment:
@Shanedell, do you happen know why this is happening?

Reply:
No I am not sure why. I am not sure if keeping track of these kinds of files is good. I know we do it daffodil-vscode with a yarn.lock to make it so there is no differences when it comes to packages but not sure if it is needed here